frontend/app-development/features/administration/components/News.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Heading, Paragraph, Label, Alert } from '@digdir/design-system-react';
+import { useNewsListQuery } from 'app-development/hooks/queries/useNewsListQuery';
+import { AltinnSpinner } from 'app-shared/components/AltinnSpinner';
+import { useTranslation } from 'react-i18next';
+import classes from './News.module.css';
+
+export const News = () => {
+  const { data: newsList, isLoading, isError } = useNewsListQuery();
+  const { t } = useTranslation();
+  return (
+    <div>
+      <Heading level={2} size='xxsmall' spacing>
+        {t('administration.news_title')}
+      </Heading>
+      <div className={classes.news}>
+        {isLoading && (
+          <AltinnSpinner spinnerText={t('general.loading')} className={classes.spinner} />
+        )}
+        {isError && <Alert severity='danger'>{t('administration.fetch_news_error_message')}</Alert>}
+        {!isLoading &&
+          !isError &&
+          newsList?.news?.map((news) => (
+            <div className={classes.newsContent} key={news.title}>
+              <Label level={3} size='small'>
+                {news.title}
+              </Label>
+              <Paragraph size='xsmall'>{news.content}</Paragraph>
+            </div>
+          ))}
+      </div>
+    </div>
+  );
+};
```

Comment:
Should we create a NewsTemplate component that can be re-used within the main render method? This would allow us to avoid checks such as !isLoading & !isError. but instead have one statement for each scenario with explicit return. 😊 

What do you think about it? 😊 

```suggestion
import * as React from 'react';
import { Heading, Paragraph, Label, Alert } from '@digdir/design-system-react';
import { useNewsListQuery } from 'app-development/hooks/queries/useNewsListQuery';
import { AltinnSpinner } from 'app-shared/components/AltinnSpinner';
import { useTranslation } from 'react-i18next';
import classes from './News.module.css';

export const News = () => {
  const { data: newsList, isLoading, isError } = useNewsListQuery();
  const { t } = useTranslation();

  if (isLoading) {
    return (
      <NewsTemplate>
        <AltinnSpinner spinnerText={t('administration.fetch_news_loading_message')} />
      </NewsTemplate>
    );
  }

  if (isError) {
    return (
      <NewsTemplate>
        <Alert severity='danger'>{t('administration.fetch_news_error_message')}</Alert>
      </NewsTemplate>
    );
  }

  return (
    <div>
      <NewsTemplate>
        {newsList.news?.map(({ title, content }) => (
          <div className={classes.newsContent} key={title}>
            <Label level={3} size='small'>
              {title}
            </Label>
            <Paragraph size='xsmall'>{content}</Paragraph>
          </div>
        ))}
      </NewsTemplate>
    </div>
  );
};

type NewsTemplateProps = {
  children: React.ReactNode;
};
const NewsTemplate = ({ children }: NewsTemplateProps) => {
  <>
    <Heading level={2} size='xxsmall' spacing>
      {t('administration.news_title')}
    </Heading>
    <div className={classes.news}>{children}</div>
  </>;
};
```

Reply:
Yes, I like it! 😄